.compose/ssl/entrypoint.sh

```
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+if [ -f rootCA.pem ] && [ -f both.pem ]; then
+  echo "Certificates already present, skipping generation."
+  exit 0
+fi
+
+mkcert -install
+mkcert $DOMAINS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)